plugins/catalog-backend/config.d.ts

```diff
@@ -163,6 +163,13 @@ export interface Config {
            * Defaults to org name if omitted.
            */
           groupNamespace?: string;
+
+          /**
+           * The namespace of the users created from this org.
+           *
+           * Defaults to empty string if omitted.
```

Comment:
Should this default to `default` or the org name instead of an empty string?

Reply:
This could be more clear-- when the string is empty, it defaults to the `default` namespace. Does this seem like ok behavior? If not, I could have it default to the org name as well.

Reply:
I think thats OK thanks for the clarification